python_ta/typecheck/base.py

```diff
@@ -100,6 +100,8 @@ def op_to_dunder_binary(op):
         return '__gt__'
     elif op == '>=':
         return '__ge__'
+    elif op == '[]':
+        return '__getitem__'
     # TODO: 'is' and 'in'
     else:
         return ''
```

Comment:
This should actually be changed. It's better to simply return `op` rather than the empty string. This will allow `_handle_call` to support arbitrary unary and binary functions.